src/store/drawing/reducer.ts

```diff
@@ -13,6 +14,8 @@ export default createReducer<DrawingState>(initialState, builder =>
         .addCase(addDrawingPathGroupAction, (state, { payload }) => {
             state.drawnPathGroups.push(payload);
         })
+        .addCase(resetCreatorAction, () => initialState) // Needed for when `PopupReply` is cancelled
```

Comment:
Should we support a flow where the user cancels their reply but wants to continue working with the existing drawing? Otherwise they'd need to start over again.

Reply:
I asked this question to product and they said that for now a cancel would start over